docs/faq.rst

```diff
@@ -78,6 +78,8 @@ Note that tool and workflow versions that do not require any file input paramete
 You can find Open Data workflows and tools by selecting the Open Data facet on the `search <https://dockstore.org/search>`_ page.
 When viewing an individual tool or workflow, you can tell which versions are Open Data on the Versions tab via the Open column.
 
+While our Open Data facet is conservative and only considers freely accessible data, other data marked as `requester pays <https://cloud.google.com/storage/docs/requester-pays>`_ may be trivial to access given payment".
```

Comment:
Suggest you do this as Note, so it sticks out a bit more.

The wording seems a little complex to me; I suggest something like Dockstore does not consider requester pays data to be open data, even though...

I believe linking to GS like that implies this is only Google Cloud thing; I suggest qualifying the link with "such as" and/or also having a link to the s3's requester pays.